veloroutes_voies_vertes/install/sql/veloroutes/20_TABLE_SEQUENCE_DEFAULT.sql

```diff
@@ -0,0 +1,573 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 10.10
+-- Dumped by pg_dump version 10.10
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+-- itineraire
+CREATE TABLE veloroutes.itineraire (
+    numero text NOT NULL,
+    nom_officiel text,
+    nom_usage text,
+    depart text,
+    arrivee text,
+    id_local integer NOT NULL,
+    annee_inscription date,
+    site_web text,
+    annee_ouverture date,
+    est_inscrit public.charbool,
+    niveau_schema text
+);
+
+
+-- ItineraireCyclable_id_seq
+CREATE SEQUENCE veloroutes."ItineraireCyclable_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+-- ItineraireCyclable_id_seq
+ALTER SEQUENCE veloroutes."ItineraireCyclable_id_seq" OWNED BY veloroutes.itineraire.id_local;
+
+
+-- liaison
+CREATE TABLE veloroutes.liaison (
+    "PRECISION" text,
+    "SRC_GEOM" text,
+    "SRC_ANNEE" text,
+    id_local integer NOT NULL,
+    id_repere integer,
+    id_poi integer
+);
+
+
+-- LiaisonCyclable_id_seq
+CREATE SEQUENCE veloroutes."LiaisonCyclable_id_seq"
```

Comment:
```suggestion
CREATE SEQUENCE veloroutes.liaison_id_local_seq
```
Problème de création de la table et du serial